tests_gpu/generators/test_optimize_acqf_generator.py

```diff
@@ -49,16 +49,20 @@
 class TestOptimizeAcqfGenerator(unittest.TestCase):
     @parameterized.expand(acqfs)
     def test_gpu_smoketest(self, acqf, acqf_kwargs):
-        lb = torch.tensor([0])
-        ub = torch.tensor([1])
+        self.device = torch.device("cuda")
+        lb = torch.tensor([0.0], device=self.device)
```

Comment:
If this is failing it is actually a problem, you shouldn't need to manually change the device before init, the tensors should always follow the model's device.

Reply:
That is true, but it does not apply to `OptimizeAcqfGenerator` as it requires bounds now, so I passed the device to it for now.